Drivers/CMSIS/Device/ST/STM32L0xx/Include/system_stm32l0xx.h

```diff
@@ -1,95 +1,97 @@
 /**
- ******************************************************************************
- * @file    system_stm32l0xx.h
- * @author  MCD Application Team
- * @brief   CMSIS Cortex-M0+ Device Peripheral Access Layer System Header File.
- ******************************************************************************
- * @attention
- *
- * <h2><center>&copy; Copyright(c) 2016 STMicroelectronics.
- * All rights reserved.</center></h2>
- *
- * This software component is licensed by ST under BSD 3-Clause license,
- * the "License"; You may not use this file except in compliance with the
- * License. You may obtain a copy of the License at:
- *                        opensource.org/licenses/BSD-3-Clause
- *
- ******************************************************************************
- */
+  ******************************************************************************
+  * @file    system_stm32l0xx.h

```

Comment:
Thanks whitespace. This is probably a clang format thing, probably same in all the HAL files. Maybe we can just make the pre-commit hook @JenishPatel99 is working on ignore files in this folder